website/content/en/docs/Demo/java-blob.md

```diff
@@ -68,7 +71,7 @@ In this step, we'll create a new user-assigned identity which will be used to in
         --scope $IDENTITY_RESOURCE_ID
```

Comment:
`$CLUSTER_MSI_CLIENTID` -> `$CLUSTER_MSI_CLIENT_ID`. Was about to do it as apart of #878 but looks like it was merged before I could get around to it.

Reply:
GitHub won't let me comment on the exact line but it's the line just about this line.

Reply:
good catch!